lxd/db/cluster/stmt.go

```diff
@@ -41,6 +41,10 @@ var stmts = map[int]string{} // Statement code to statement SQL text.
 // PreparedStmts is a placeholder for transitioning to package-scoped transaction functions.
 var PreparedStmts = map[int]*sql.Stmt{}
 
+// DqliteMaxParams is the maximum amount of SQL query parameters that are supported by dqlite.
+// See: https://github.com/canonical/dqlite/issues/377
+const DqliteMaxParams = 255
```

Comment:
You can use this directly from the driver https://github.com/canonical/go-dqlite/pull/194/files#diff-bd895b31b8e89a49fd8420f2fbdd9bfc78046bd2eefd9c5f5b848c643e9c8d9bR64

Reply:
Oh cool, thanks for that!